library/std/src/sys/personality/dwarf/mod.rs

```diff
@@ -17,32 +17,30 @@ pub struct DwarfReader {
     pub ptr: *const u8,
 }
 
-#[repr(C, packed)]
-struct Unaligned<T>(T);
-
+#[deny(unsafe_op_in_unsafe_fn)]
```

Comment:
Please use forbid so that we know this can't be reallowed inside.
```suggestion
#[forbid(unsafe_op_in_unsafe_fn)]
```

Reply:
oh, sure.

currently my usage of nested deny/allow/deny is _in general_ intentional, but yeah this is a small enough scope that we might as well forbid it.